src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -2164,6 +2173,12 @@ static void uct_ib_mlx5dv_check_dm_ksm_reg(uct_ib_mlx5_md_t *md)
                  ucs_status_string(status));
         return;
     }
+
+    ucs_debug("%s: KSM over device memory is supported",
+              uct_ib_device_name(&md->super.dev));
```

Comment:
1. "%s: device memory allocation is supported"
2. move this log after line 2167? we could get here also if uct_ib_mlx5_devx_reg_atomic_key failed